packages/paypal-commerce-integration/src/paypal-commerce-alternative-methods/paypal-commerce-ratepay-payment-strategy.ts

```diff
@@ -194,6 +196,16 @@ export default class PaypalCommerceRatepayPaymentStrategy implements PaymentStra
         return `${date.getFullYear()}-${formattedMonth}-${formattedDate}`;
     }
 
+    private onPending(isPending: boolean) {
+        const { onPending } = this.paypalcommerceratepay || {};
+
+        if (!onPending || typeof onPending !== 'function') {
+            return;
+        }
+
+        onPending(isPending);
```

Comment:
This seems quite odd, we have a private method called `onPending`, which inside gets `onPending` and calls `onPending`.

Can we call the value on `paypalcommerceratepay` initialize options to be something like `onPaymentSubmission`? A name which makes more sense to what is actually going on.